stdlib/src/bz2.rs

```diff
@@ -12,28 +12,48 @@ mod _bz2 {
         object::{PyPayload, PyResult},
         types::Constructor,
     };
+    use crate::zlib::{
```

Comment:
Since there is at-least 1 other decompression module that uses the same format (`_lzma`). I think this should be moved someplace common.

Reply:
I agree - I was planning on doing that in a followup, to reduce the amount of code movement in this pr.